envoy/network/connection_handler.h

```diff
@@ -17,6 +17,28 @@
 namespace Envoy {
 namespace Network {
 
+// An effectively abstract class, used for dynamic class inheritance via dynamic_cast
+// such that options specific to individual listener types can be passed in
+// through the shutdownListener function, without other listener types needing
+// to know about it. Subclasses should specify `virtual ExtraShutdownListenerOptions`
+// so that they can be combined with other subclasses into a class containing multiple
+// types of options.
+class ExtraShutdownListenerOptions {
+public:
+  virtual ~ExtraShutdownListenerOptions() = default;
+};
+
+// An interface for ExtraShutdownListenerOptions enabling us to forward UDP packets from
```

Comment:
nit: Avoid first person in comments since it can be a bit ambiguous who this refers to. In this case, How about: "enabling Envoy" or "enabling the Listener".

Reply:
Changed.